jumpcloud-ADMU/Powershell/Tests/manualTests/selectionForm.feature

```diff
@@ -0,0 +1,217 @@
+        And the "Install JCAgent" checkbox is selected
+        And a "11111111111111111111 1111111111111111111" is pasted in the ConnectKey text field
+        And a JumpCloud valid username is specified in the username text box
+        And a valid password is specified in the password text box
+        Then the "Migrate Profile" button should NOT become active and be greyed-out
+
+Feature: JumpCloud Username & AutoBind User Validation
+
+    Scenario: Disallows migration for usernames that are not found in the JumpCloud organization
+        Given the JumpCloud agent is NOT installed on the device
+        And The selection form is opened
+        And a domain (local AD or EntraID) user has logged into the device
+        When a domain profile is selected
+        And the "AutoBind JCUser" checkbox is selected
+        And a valid API Key is pasted in the APIKey text field
+        And a JumpCloud username that does or does not exist in the JumpCloud Organization is specified in the username text box
+        And a valid password is specified in the password text box
+        And the "Migrate Profile" button is pressed
+        Then a windows form window should appear and warn: The JumpCloud agent is not installed and to also enter your connectKey
+
+    Scenario: Disallows migration for usernames that are not found in the JumpCloud organization
+        Given the JumpCloud agent is installed on the device
+        And The selection form is opened
+        And a domain (local AD or EntraID) user has logged into the device
+        When a domain profile is selected
+        And the "AutoBind JCUser" checkbox is selected
+        And a valid API Key is pasted in the APIKey text field
+        And a JumpCloud username that does not exist in the JumpCloud Organization is specified in the username text box
+        And a valid password is specified in the password text box
+        And the "Migrate Profile" button is pressed
+        Then a windows form window should appear and note the specified username was not found in the JumpCloud Organization
+
+    Scenario: Prompts warning for migration for usernames that have a localSystemUsername specified in the JumpCloud Organization
+        Given the JumpCloud agent is installed on the device
+        And The selection form is opened
+        And a domain (local AD or EntraID) user has logged into the device
+        When a domain profile is selected
+        And the "AutoBind JCUser" checkbox is selected
+        And a valid API Key is pasted in the APIKey text field
+        And a JumpCloud username that does exist in the JumpCloud Organization is specified in the username text box
+        And a valid password is specified in the password text box
+        And the "Migrate Profile" button is pressed
+        Then a windows form window should appear and note that the specified user has a local account username/ can select OK or Cancel to return to selection form
```

Comment:
Form window seems to not show the right message as seen with the last 2 tests: 
<img width="747" alt="image" src="https://github.com/user-attachments/assets/c53a878e-4049-4514-972e-88898f590ed7" />

Reply:
disregard, form shows the correct message